buildconfig/pygame-stubs/sprite.pyi

```diff
@@ -72,58 +55,48 @@ class RenderClear(Group):
-    def draw(self, surface: Surface) -> List[Rect]: ...
-    def get_sprites_at(
-        self, pos: Union[Tuple[int, int], List[int]]
-    ) -> List[Sprite]: ...
+    def add(self, *sprites: Sprite, **kwargs: int) -> None: ...
+    def draw(self, surface: Surface) -> Sequence[Rect]: ...
+    def get_sprites_at(self, pos: Union[Tuple[int, int], Sequence[int]]) -> Sequence[Sprite]: ...
     def get_sprite(self, idx: int) -> Sprite: ...
-    def remove_sprites_of_layer(self, layer_nr: int) -> List[Sprite]: ...
-    def layers(self) -> List[int]: ...
+    def remove_sprites_of_layer(self, layer_nr: int) -> Sequence[Sprite]: ...
+    def layers(self) -> Sequence[int]: ...
     def change_layer(self, sprite: Sprite, new_layer: int) -> None: ...
     def get_layer_of_sprite(self, sprite: Sprite) -> int: ...
     def get_top_layer(self) -> int: ...
     def get_bottom_layer(self) -> int: ...
     def move_to_front(self, sprite: Sprite) -> None: ...
     def move_to_back(self, sprite: Sprite) -> None: ...
     def get_top_sprite(self) -> Sprite: ...
-    def get_sprites_from_layer(self, layer: int) -> List[Sprite]: ...
-    def switch_layer(self, layer1_nr, layer2_nr) -> None: ...
+    def get_sprites_from_layer(self, layer: int) -> Sequence[Sprite]: ...
+    def switch_layer(self, layer1_nr: int, layer2_nr: int) -> None: ...
 
 class LayeredDirty(LayeredUpdates):
-    def __init__(self, *sprites: DirtySprite, **kwargs: Dict[str, Any]):
-        LayeredUpdates.__init__(self, *sprites, **kwargs)
+    def __init__(self, *sprites: DirtySprite, **kwargs: int) -> None: ...
     def copy(self) -> LayeredDirty: ...
-    def draw(self, surface: Surface, bgd: Optional[Surface] = None) -> List[Rect]: ...
+    def draw(self, surface: Surface, bgd: Optional[Surface] = ...) -> Sequence[Rect]: ...
     def clear(self, surface: Surface, bgd: Surface) -> None: ...
     def repaint_rect(self, screen_rect: _RectStyle) -> None: ...
-    def set_clip(self, screen_rect: Optional[_RectStyle] = None): ...
+    def set_clip(self, screen_rect: Optional[_RectStyle] = ...) -> None: ...  # TODO: Verify
     def get_clip(self) -> Rect: ...
-    def set_timing_treshold(
-        self, time_ms: SupportsFloat
-    ) -> None: ...  # This actually accept any value
+    def set_timing_treshold(self, time_ms: SupportsFloat) -> None: ...  # This actually accept any value
 
 class GroupSingle(AbstractGroup):
     sprite: Sprite
-    def __init__(self, sprite: Optional[Sprite]):
-        AbstractGroup.__init__(self)
+    def __init__(self, sprite: Optional[Sprite] = ...) -> None: ...
     def copy(self) -> GroupSingle: ...
 
 def spritecollide(
-    sprite: Sprite,
-    group: AbstractGroup,
-    dokill: bool,
-    collided: Optional[Callable[[Sprite, Sprite], bool]] = None,
-) -> List[Sprite]: ...
+    sprite: Sprite, group: AbstractGroup, dokill: bool, collided: Optional[Callable[[Sprite, Sprite], bool]] = ...,
+) -> Sequence[Sprite]: ...
```

Comment:
Not sure this should be Sequence when it returns List.

Reply:
made it a list